### DISCUSSION TITLE
Elements which added by dom API

### DISCUSSION THREAD

**Discussion by YousofSayed** [GrapesJS/grapesjs#6309] [Q&A]
Hello , 
Really special thanks for your effort 💙,
Now i wanna to make editor automatically load element which added by dom API like `(el.appendChild(child))` ,
How can i do that or there is any built in option to handle that ?
thanks🌹
@artf

**Comment by danstarns** [comment]
Hi @YousofSayed, 

This is a little hard to understand, please could you rephrase and or produce a reproductive explanation of what you need GrapesJS to do. Thank you 🍇
